<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Server</class>
 <widget class="QDialog" name="Server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>发送</string>
  </property>
  <widget class="QLabel" name="server_status_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>请选择要发送的文件！</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="server_progressBar">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>261</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="server_open_btn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>打开...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="server_send_btn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="server_close_btn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QLabel" name="server_choose_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>选择文件/文件夹</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
